### ISSUE TITLE
Additional email accounts [Feature Request]

### ISSUE THREAD

**Issue by versavius** [opensupports/opensupports#350]
We currently can have one email account where staff can email to and emails sent from for the app.

Say my company has three departments:

IT (it@example.com)
HR (hr@example.com)
Sales (sales@example.com)

What is required is to be able to have each department to have one email address assigned to it

This allows multiple departments use one installation of this app, so you can give our different email addresses to different people, whether internal or external so you email IT it goes into the correct department group.

Same with HR and Sales.

This is important so that tickets get assigned to the correct group and you dont need to give everyone access to all departments.

**Comment by nickbe** [comment]
This seems rather unusual for an issue tracker. Usually you receive any email from one sender. noreply@ or support@.

**Comment by versavius** [comment]
Yes and no, most of the time yes you would use one email address for everything but i have seen some organisations use systems like osticket and others which can manage this.

The advantage is that you can have all your company departments using one support portal so if tickets need to be passed between departments which can easily happen between IT and HR you simply re-assign the ticket to the department and do not need multiple installations of the application for each app.

**Comment by nickbe** [comment]
But reassigning tickets between departments can already be done as it is. Systems like osTicket can do a lot more, but that is exactly the point why we stopped using their kind in the first place. Too complicated to maintain for a small company, too clumsy to handle and rejected by our clients for being ugly and too much.
OpenSupports is simple, and although the team is trying to make some enhancements it's best they also try to keep it as simple as possible.

**Comment by mredigonda** [comment]
I think any kind of rule based system would be too complicated and as nickbe points out, it would break OpenSupports' simplicity.

However, having departments connected to specific email accounts sounds useful. So far departments have almost no info associated with them (just a boolean value for private/public), and I see no harm on adding an email field.

This is just my personal opinion on the topic.

**Comment by versavius** [comment]
Yes that would work so say you have

Finance = finance@xxx.com
IT Support = itsupport@xxx.com

etc so when an email is sent to a particular department it gets populated in the correct department group

**Comment by nickbe** [comment]
Sounds clever and simple enough 👍
